Client/Frontend/Widgets/LoginTableViewCell.swift

```diff
@@ -52,10 +53,14 @@ class LoginTableViewCell: UITableViewCell {
         label.autocorrectionType = .No
         label.accessibilityElementsHidden = true
         label.adjustsFontSizeToFitWidth = false
+        label.delegate = self
         return label
     }()
 
-    lazy var highlightedLabel: UILabel = {
+    // Exposing this label as internal/public causes the Xcode 7.2.1 compiler optimizer to
+    // produce a EX_BAD_ACCESS error when dequeuing the cell. For now, this label is made private
+    // and the text property is exposed using a get/set property below.
+    private lazy var highlightedLabel: UILabel = {
```

Comment:
I wonder if keeping it public but making this non-`lazy` would also fix the issue? If that worked, you wouldn't have to change as much in the PR, but this is fine too.